jbpm-test-coverage/src/test/resources/org/jbpm/test/functional/task/buildPersonDecision.dmn

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" ?>
+<dmn:definitions
+	xmlns:dmn="http://www.omg.org/spec/DMN/20180521/MODEL/"
+	xmlns="https://kiegroup.org/dmn/_DBBCAFFF-161B-4D7A-809B-320EDF3933FC"
+	xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/"
+	xmlns:kie="http://www.drools.org/kie/dmn/1.2"
+	xmlns:feel="http://www.omg.org/spec/DMN/20180521/FEEL/"
+	xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/"
+	xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" id="_972F1166-02C5-4389-8405-224781EFCF6A" name="simple" expressionLanguage="http://www.omg.org/spec/DMN/20180521/FEEL/" typeLanguage="http://www.omg.org/spec/DMN/20180521/FEEL/" namespace="https://kiegroup.org/dmn/_DBBCAFFF-161B-4D7A-809B-320EDF3933FC">
+	<dmn:extensionElements/>
+	<dmn:itemDefinition id="_B7DC43AB-5DD9-4CFC-93F9-A9F3F563C0AB" name="Person" isCollection="false">
+		<dmn:itemComponent id="_A8E45253-E12B-4CB8-80EE-BB2DACDA8C6C" name="age" isCollection="false">
+			<dmn:typeRef>number</dmn:typeRef>
+		</dmn:itemComponent>
+		<dmn:itemComponent id="_8097D6E0-01DD-410D-A189-8D69704654F2" name="name" isCollection="false">
+			<dmn:typeRef>string</dmn:typeRef>
+		</dmn:itemComponent>
+	</dmn:itemDefinition>
+	<dmn:decision id="_F3595CA0-6666-4CF3-A81A-D21870D95379" name="buildPersonDecision">
+		<dmn:extensionElements/>
+		<dmn:variable id="_828DED89-ADA3-4973-8469-95C564DF06BD" name="personBRL" typeRef="Person"/>
```

Comment:
I'm not sure how you ended up with the DMN model content, but it's a bit strange to me that variable and Decision are not sharing the same name?
Normally the editor would assign it the same name
can you confirm?

Reply:
I notice that the name of the outcome is the name of the variable and not the name of the decision. So for testing this properly I set them with different names.

Reply:
I don't see why you should
DMN expects usually 
for the Decision name = Variable name
every DRGElement name = inherent variable name